data/ignore_ids_safety_scan.json

```diff
@@ -0,0 +1,138 @@
+{
+    "tensorflow": {
+        "training": {
+            "_comment":"py2 is deprecated",
+            "py2": {
+                "38449":"for shipping pillow<=6.2.2 - the last available version for py2",
+                "38450":"for shipping pillow<=6.2.2 - the last available version for py2",
+                "38451":"for shipping pillow<=6.2.2 - the last available version for py2",
+                "38452":"for shipping pillow<=6.2.2 - the last available version for py2",
+                "35015":"for shipping pycrypto<=2.6.1 - the last available version for py2"
+            },
+            "py3": {
```

Comment:
Should this contain the python minor version as well? What happens when a DLC has 2 python version support?

Reply:
Synced up